.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,71 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
+// README at: https://github.com/devcontainers/templates/tree/main/src/typescript-node
+{
+  "name": "Radashi",
+  // Or use a Dockerfile or Docker Compose file. More info: https://containers.dev/guide/dockerfile
+  "image": "mcr.microsoft.com/devcontainers/typescript-node:1-20-bookworm",
+
+  // Features to add to the dev container. More info: https://containers.dev/features.
+  "features": {
+    "ghcr.io/devcontainers-contrib/features/pnpm": {
+      "version": "9.1.3"
+    },
+    "ghcr.io/joshuanianji/devcontainer-features/mount-pnpm-store": {}
+  },
+  "postCreateCommand": "pnpm i",
+  "customizations": {
+    "vscode": {
+      "extensions": [
+        "vitest.explorer",
+        "biomejs.biome",
+        "esbenp.prettier-vscode"
+      ],
+      "settings": {
+        "editor.formatOnSave": true,
+        "editor.codeActionsOnSave": {
+          "source.organizeImports": "explicit",
+          "quickfix.biome": "explicit"
+        },
+        "typescript.tsdk": "./node_modules/typescript/lib",
+        "typescript.enablePromptUseWorkspaceTsdk": true,
+        "typescript.preferences.importModuleSpecifier": "non-relative",
+        "typescript.preferences.autoImportFileExcludePatterns": [
+          "**/node_modules/**",
+          "**/dist/**",
+          "**/dist"
+        ],
+        "[markdown]": {
+          "editor.defaultFormatter": "esbenp.prettier-vscode"
+        },
+        "[mdx]": {
+          "editor.defaultFormatter": "esbenp.prettier-vscode"
+        },
+        "[json]": {
+          "editor.defaultFormatter": "biomejs.biome"
+        },
+        "[jsonc]": {
+          "editor.defaultFormatter": "biomejs.biome"
+        },
+        "[typescript]": {
+          "editor.defaultFormatter": "biomejs.biome",
+          "rewrap.wrappingColumn": 70
+        },
+        "[shellscript]": {
+          "rewrap.wrappingColumn": 70
+        }
+      }
+    }
+  }
```

Comment:
is this actually necessary? do dev containers not use `.vscode/settings.json`?

Reply:
the extensions part, yes, because this way when creating the development container they are already installed 😃. but the rest is really an unnecessary duplication.
I think it was my force of habit to leave only the configurations in the dev container.
I'm going to change this